simplic-change-tracking/Simplic.Change.Tracking/Interfaces/IChangeTrackingRepository.cs

```diff
@@ -13,7 +13,7 @@ public interface IChangeTrackingRepository
         ChangeTracking Get(Int64 id);
         IEnumerable<ChangeTracking> GetChanges(object primaryKey);
         byte[] GetJsonAsByteArray(long ident);
-        IEnumerable<ChangeTracking> GetChangesWithObject(object poco, string dataColumn = "");
+        IEnumerable<ChangeTracking> GetChangesWithObject(ChangeTrackingKey poco, string dataColumn = "");
```

Comment:
`ChangeTrackingKey` --> `changeTrackingKey`. Always add comments to the interface as well.